### PR TITLE
Add Import Devices in Bulk - resolves enhancement #55

### PR DESCRIPTION
**Pull request by kuba-mazurkiewicz**
Add Import Devices in bulk /dna/intent/api/v1/onboarding/pnp-device/import

### CODE REVIEW

File: gen/definitions/pnp_import_devices.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: PnP Import Devices
+rest_endpoint: /dna/intent/api/v1/onboarding/pnp-device/import
+res_description: 'This resource adds devices to PNP in bulk  file on the chosen network device (upgrade the software on the device).
```
Comment (danischm) [inline gen/definitions/pnp_import_devices.yaml:4]: The first line does not make sense..

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/pnp_import_devices.yaml (hunk 1 above)
Comment (kuba-mazurkiewicz) [reply gen/definitions/pnp_import_devices.yaml]: sorry, my mistake, fix that

File: gen/definitions/pnp_import_devices.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: PnP Import Devices
 rest_endpoint: /dna/intent/api/v1/onboarding/pnp-device/import
-res_description: 'This resource adds devices to PNP in bulk  file on the chosen network device (upgrade the software on the device).
+res_description: 'This resource adds devices to PNP in bulk.
   Every time this resource is created or re-created, the Catalyst Center considers adding new devices to PNP.
   When this resource is destroyed or updated or refreshed, no actions are done either on CatalystCenter or on devices'
```
Comment (danischm) [inline gen/definitions/pnp_import_devices.yaml:6]: Can we add a dot at the end of the sentence? :)

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/pnp_import_devices.yaml (hunk 2 above)
Comment (kuba-mazurkiewicz) [reply gen/definitions/pnp_import_devices.yaml]: done

**Review by danischm — APPROVED** [review]

**Pull request merged**